src/components/Landing/index.tsx

```diff
@@ -177,7 +182,10 @@ export default function Landing() {
                       <Button
                         type="primary"
                         size="large"
-                        onClick={scrollToCreate}
+                        onClick={() => {
+                          fathom.trackGoal('IIYVJKNC', 0)
```

Comment:
Track clicks on this button, for example.

This is blocked by ad-blockers.

Reply:
Would be dope to add some type safety one we get more `event-id`s but this looks solid.

Reply:
Yeah yeah for sure.